web/app/scripts/views/map/layers-controller.js

```diff
@@ -771,7 +781,7 @@
         }
 
         function getTilekeys() {
-            var primary = QueryBuilder.djangoQuery(0, getAdditionalParams()).then(
+            var primary = QueryBuilder.djangoQuery(0, getAdditionalParams(), true, true, false).then(
```

Comment:
I think we would want a details-only query here, but actually I don't think it would make a difference.  The tilekey saves the SQL query, which windshaft uses to get the records directly, so the special serializer handling doesn't come into play.

This would also explain why I'm seeing one 10MB tile in my network log...